content/en/docs/concepts/scheduling-eviction/node-pressure-eviction.md

```diff
@@ -275,8 +275,35 @@ will choose to evict pods of lowest Priority first.
 
 If you are running a [static pod](/docs/concepts/workloads/pods/#static-pods)
 and want to avoid having it evicted under resource pressure, set the
-`priority` field for that Pod directly. Static pods do not support the
-`priorityClassName` field.
+`priorityClassName` field for that Pod directly. 
+
+For example, the following configuration sets the priorityClassName for a static pod as `xyz-priority`. 
+
+```yaml
+kubectl get priorityclass
+NAME                      VALUE        GLOBAL-DEFAULT   AGE
+system-cluster-critical   2000000000   false            2m2s
+system-node-critical      2000001000   false            2m2s
+xyz-priority              1000000      false            47s
+```
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: static-web
+  labels:
+    role: myrole
+spec:
+  containers:
+    - name: web
+      image: nginx
+      ports:
+        - name: web
+          containerPort: 80
+          protocol: TCP
+  priorityClassName: xyz-priority
```

Comment:
This will be fine with one extra caveat: if your static Pod could ever start on a cluster that doesn't yet have a running control plane, the kubelet won't be able to resolve the PriorityClass.

This is important because running the control plane is likely the most common use for static pods, and people probably do want to protect those from eviction.

Reply:
The work-around is to _also_ define `priority` for any static Pod that might need it.

Reply:
When I set priorityClass only, the priority is displayed in the description of the pod.  Are you stating to set the priority number higher than any others, for this example?

Reply:
Only setting priority fails, and is the original reason for this issue.

Reply:
Sure. You have to set **both** for any static Pod that might start before there's a healthy control plane. The existing docs are wrong; I stand by https://github.com/kubernetes/website/pull/47055#discussion_r1663601731 though.

(and, TBH, the way the admission control mechanism works is unhelpful, but we need to document how K8s actually works).